dali/operators/imgcodec/decoder_test_helper.h

```diff
@@ -238,7 +238,7 @@ inline void AssertClose(const Tensor<CPUBackend> &img, const Tensor<CPUBackend>
 }
 
 inline Tensor<CPUBackend> ReadReference(InputStream *src, TensorLayout layout = "HWC") {
-  auto tensor = numpy::ReadTensor(src);
+  auto tensor = numpy::ReadTensor(src, true);
```

Comment:
```suggestion
  auto tensor = numpy::ReadTensor(src, false);
```
I've checked and it looks like this function is used only in tests that don't involve the GPU.

Reply:
Sure, I put `true` just to maintain the default value it had before introducing the `bool pinned` argument :+1: